test/Microsoft.SqlTools.ServiceLayer.UnitTests/NotebookConvert/NotebookConvertServiceTests.cs

```diff
@@ -90,16 +87,82 @@ FROM sys.databases
       ""cell_type"": ""markdown"",
       ""source"": [
         ""* Ending multiline  \n"",
-        "" * comment""
+        ""* comment""
       ]
     }
   ]
 }";
+        [Test]
+        public void ConvertSqlToNotebook()
+        {
+            var notebook = NotebookConvertService.ConvertSqlToNotebook(sampleSqlQuery);
+            var notebookString = JsonConvert.SerializeObject(notebook, Formatting.Indented);
+            Assert.AreEqual(sampleNotebook, notebookString);
+        }
 
-            var notebook = NotebookConvertService.ConvertSqlToNotebook(sql);
+        [Test]
+        public void ConvertNullSqlToNotebook()
+        {
+            var emptyNotebook = @"{
+  ""metadata"": {
+    ""kernelspec"": {
+      ""name"": ""SQL"",
+      ""display_name"": ""SQL"",
+      ""language"": ""sql""
+    },
+    ""language_info"": {
+      ""name"": ""sql"",
+      ""version"": """"
+    }
+  },
+  ""nbformat_minor"": 2,
+  ""nbformat"": 4,
+  ""cells"": []
+}";
+            var notebook = NotebookConvertService.ConvertSqlToNotebook(null);
             var notebookString = JsonConvert.SerializeObject(notebook, Formatting.Indented);
-            Assert.AreEqual(expectedNotebook, notebookString);
+            Assert.AreEqual(emptyNotebook, notebookString);
+        }
+
+        [Test]
+        public void ConvertNotebookToSql()
+        {
+            var expectedSqlQuery = @"/*
+* Initial multiline comment
+*/
+
+/*
+Comment before batch
+*/
+
+SELECT * FROM sys.databases
+
+-- Compare Row Counts in Tables From Two Different Databases With the Same Schema
+
+SELECT * -- inline single line comment
+/* inline multiline
+ * comment
+ */
+FROM sys.databases
+
+/*
+ending single line comment
+*/
+
+/*
+* Ending multiline  
+* comment
+*/";
+            var notebook = JsonConvert.DeserializeObject<NotebookDocument>(sampleNotebook);
+            var query = NotebookConvertService.ConvertNotebookDocToSql(notebook);
+            Assert.AreEqual(expectedSqlQuery, query);
```

Comment:
We should be trying to use NUnit fluent assertions whenever possible `Assert.That(query, Is.EqualTo(expectedSqlQuery))`